.gitignore

```diff
@@ -0,0 +1 @@
+_temp
```

Comment:
Why `_temp` here rather than `_bin` which is used in other projects?

Reply:
For no reason other than to prove that this works too.

Reply:
done